rero_ils/modules/acq_orders/jsonschemas/acq_orders/acq_order-v0.0.1.json

```diff
@@ -0,0 +1,235 @@
+          {
+            "value": "monograph",
+            "label": "Monograph"
+          },
+          {
+            "value": "serial",
+            "label": "Serial"
+          },
+          {
+            "value": "standing order",
+            "label": "Standing order"
+          },
+          {
+            "value": "monographic set",
+            "label": "Monographic set"
+          },
+          {
+            "value": "planned order",
+            "label": "Planned order"
+          },
+          {
+            "value": "multi-volume",
+            "label": "Multi-volume"
+          }
+        ]
+      }
+    },
+    "description": {
+      "title": "Description",
+      "type": "string",
+      "minLength": 5
+    },
+    "order_status": {
+      "title": "Order status",
+      "type": "string",
+      "enum": [
+        "approved",
+        "canceled",
+        "ordered",
+        "requested",
+        "pending",
+        "received"
+      ],
+      "default": "pending",
+      "form": {
+        "options": [
+          {
+            "value": "approved",
+            "label": "Approved"
+          },
+          {
+            "value": "canceled",
+            "label": "Canceled"
+          },
+          {
+            "value": "ordered",
+            "label": "Ordered"
+          },
+          {
+            "value": "requested",
+            "label": "Requested"
+          },
+          {
+            "value": "pending",
+            "label": "Pending"
+          },
+          {
+            "value": "received",
+            "label": "Received"
+          }
+        ]
+      }
+    },
+    "currency": {
+      "title": "Order currency",
+      "type": "string",
+      "enum": [
+        "CHF",
+        "EUR",
+        "USD"
+      ],
+      "default": "CHF"
+    },
+    "order_date": {
+      "title": "Order date",
+      "type": "string",
+      "format": "date",
+      "pattern": "\\d{4}-((0[1-9])|(1[0-2]))-(((0[1-9])|[1-2][0-9])|(3[0-1]))$",
+      "validationMessage": "Required. Should be in the ISO 8601, YYYY-MM-DD.",
+      "form": {
+        "type": "datepicker",
+        "placeholder": "Choose a date",
+        "wrappers": [
+          "form-field"
+        ],
+        "validation": {
+          "messages": {
+            "pattern": "Required. Should be in the ISO 8601, YYYY-MM-DD."
+          }
+        }
+      }
+    },
+    "total_amount": {
+      "title": "Total amount",
+      "type": "number",
+      "minLength": 1,
+      "default": 0
+    },
+    "vendor": {
+      "title": "Vendor",
+      "type": "object",
+      "required": [
+        "$ref"
+      ],
+      "properties": {
+        "$ref": {
+          "title": "Vendor URI",
+          "type": "string",
+          "pattern": "^https://ils.rero.ch/api/vendors/.*?$",
+          "form": {
+            "focus": true,
+            "remoteOptions": {
+              "type": "vendors"
+            },
+            "placeholder": "Choose a vendor",
```

Comment:
Is placehloder property works with a select?

Reply:
yes, it works